wit-0.3.0-draft/system-clock.wit

```diff
@@ -1,27 +1,34 @@
 package wasi:clocks@0.3.0;
-/// WASI Wall Clock is a clock API intended to let users query the current
-/// time. The name "wall" makes an analogy to a "clock on the wall", which
-/// is not necessarily monotonic as it may be reset.
+/// WASI System Clock is a clock API intended to let users query the current
+/// time. The clock is not necessarily monotonic as it may be reset.
 ///
 /// It is intended to be portable at least between Unix-family platforms and
 /// Windows.
 ///
-/// A wall clock is a clock which measures the date and time according to
-/// some external reference.
+/// An "instant", or "exact time", is a point in time without regard to any time
+/// zone: just the time since a particular external reference point, often
+/// called an "epoch".
 ///
 /// External references may be reset, so this clock is not necessarily
 /// monotonic, making it unsuitable for measuring elapsed time.
 ///
 /// It is intended for reporting the current date and time for humans.
 @since(version = 0.3.0)
-interface wall-clock {
-    /// A time and date in seconds plus nanoseconds.
+interface system-clock {
+    /// An exact time in seconds plus nanoseconds.
```

Comment:
Could you add a URL linking to the Temporal documentation defining "exact time"?

Reply:
Added a link to https://tc39.es/proposal-temporal/docs/timezone.html.